### PR TITLE
feat: loop UI & lobby improvements

### PR DESCRIPTION
**Pull request by jstarpl**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This is a new feature/improvement.

* **What is the current behavior?** (You can also link to an open issue here)

Some of the looping information is displayed inconsistently or inconveniently. The looping UI was a quick implementation, just to check if the backend works.

* **What is the new behavior (if this is a feature change)?**

The way looping information is displayed in the Lobby has been improved. The way the information in the Rundown View is conveyed has also been improved. Some minor tweaks to the Lobby layout/typography/coloring have been made to make it more consistent overall.

Lobby
Rundown View

* **Other information**:

**Status**

- [x] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [x] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/client/ui/RundownList.tsx [hunk 1]
```diff
@@ -298,14 +298,13 @@ export const RundownList = translateWithTracker(() => {
 												</Tooltip>
 											</span>
 											{/* <span className="rundown-list-item__problems">{t('Problems')}</span> */}
-											<span>{t('Show style')}</span>
+											<span>{t('Show Style')}</span>
```
Comment (gundelsby) [inline meteor/client/ui/RundownList.tsx:301]: Will this invalidate the translation keys, and thus also the existing translation?

File: meteor/client/ui/RundownList.tsx [hunk 2]
```diff
@@ -298,14 +298,13 @@ export const RundownList = translateWithTracker(() => {
...
 											<span>{t('On Air Start Time')}</span>
 											<span>{t('Duration')}</span>
-											<span>{t('Last updated')}</span>
+											<span>{t('Last Updated')}</span>
```
Comment (gundelsby) [inline meteor/client/ui/RundownList.tsx:304]: Same here

**Review by gundelsby — APPROVED** [review]
LGTM, I assume this won't be first time we've changed a text and the translations would have to be updated anyway :)

File: meteor/client/ui/RundownList.tsx (hunk 1 above)
Comment (jstarpl) [reply meteor/client/ui/RundownList.tsx:301]: Yeah... "Such Is Life" 🇱🇻

**Pull request merged**